tcms/testplans/templates/testplans/search.html

```diff
@@ -125,6 +125,10 @@
     </div>
 </div><!-- /container -->
 
+
+<!-- send default page size to testplans/js/search.js-->
```

Comment:
Comment is redundant, the name/id of the element makes it clear what it does, no need to repeat